index.js

```diff
@@ -60,8 +60,12 @@ function prettyTmpDirName (obj, prop) {
       .replace(/^_+/, '')
   }
 
-  var cleanObjectName = cleanString(obj.constructor && obj.constructor.name)
-  if (cleanObjectName === 'object') cleanObjectName = ''
+  var pluginName = obj.pluginName;
+  var cleanObjectName =
+      cleanString(typeof pluginName === 'string'
+          ? pluginName
+          : (obj.constructor && obj.constructor.name));
```

Comment:
I just prefer editing semicolon-less code, so all my modules don't use semicolons, but I don't feel very strongly about it. ;-)